packages/web/src/scss/settings/_utilities.scss

```
@@ -1,25 +1,49 @@
 @use 'sass:map';
 @use '@tokens' as tokens;
+@use '../tools/utilities' as tools-utilities;
+
+$accent-background-scope-list: background-basic, background-subtle;
+$accent-border-color-scope-list: border-basic, border-subtle;
+$accent-text-color-scope-list: content-basic, content-subtle;
+
+$accent-prefix: 'accent-';
+$accent-tokens-key: 'accent-colors';
 
 $utilities: (
     'background-color': (
         responsive: false,
         property: background-color,
         class: bg,
-        values: (
-            primary: tokens.$background-primary,
-            secondary: tokens.$background-secondary,
-            tertiary: tokens.$background-tertiary,
-        ),
+        values: map.merge(
+                tools-utilities.generate-token-map(
+                    $accent-tokens-key,
```

Lately, we have been mostly using named parameters, and I think I would prefer them here as well. Can you please add it?

```suggestion
                    $tokens-key: $accent-tokens-key,
                    ...
```